deploy/Makefile

```diff
@@ -1,5 +1,5 @@
 CONFIG_FILE?=./deploy.json
-TARGETS?=$(shell find template -type file | sed 's/template/target/')
+TARGETS?=$(shell find template -type f | sed 's/template/target/')
```

Comment:
I hope it works on Mac too...

Otherwise we'll have to dig up some old make magic :)